internal/console/scan.go

```diff
@@ -55,7 +56,7 @@ var scanCmd = &cobra.Command{
 
 func initializeConfig(cmd *cobra.Command) error {
 	if cfgFile == "" {
-		_, err := os.Stat(filepath.ToSlash(filepath.Join(path, "kics.config")))
+		_, err := os.Stat(filepath.ToSlash(filepath.Join(filepath.Dir(path), "kics.config")))
```

Comment:
related with json support or just slipped right in?

Reply:
related to a bug with config file